modules/UI/videolayout/ConnectionIndicator.js

```diff
@@ -198,6 +198,9 @@ ConnectionIndicator.prototype.generateText = function () {
                 }
             }
 
+            // All of the transports should be either P2P or JVB
+            const isP2P = this.transport.length ? this.transport[0].p2p : false;
```

Comment:
and fixing only this line will not help because whole code around here does not care